assets/ts/stop/components/amenities/AccessibilityAmenityCard.tsx

```diff
@@ -82,17 +82,21 @@ const AccessibilityAmenityCard = ({
   const stationOrStop = isStation ? "station" : "stop";
   const features = without(accessibleFeatures, "accessible");
   const hasAccessibleFeatures = accessibleFeatures.includes("accessible");
-  const badge = !hasAccessibleFeatures && (
+
+  const accessibleOverride = document.querySelector("div[data-mbta-accessible]")?.getAttribute("data-mbta-accessible") === "true";
```

Comment:
📓 This is kind of a hacky way of getting this information into React without having to duplicate the logic that looks at both the stop and the routes for that stop...the latter of which we don't even have in this React code.